server/bleep/src/db.rs

```diff
@@ -9,8 +9,7 @@ use crate::Configuration;
 pub type SqlDb = Arc<SqlitePool>;
 
 pub async fn init(config: &Configuration) -> Result<SqlitePool> {
-    fs::create_dir_all(&config.data_dir)?;
-    let data_dir = config.data_dir.to_string_lossy();
+    let data_dir = config.index_dir.to_string_lossy();
```

Comment:
Why remove `create_dir_all`?

Reply:
Since `data_dir` is redundant, we can rely on `index_dir` existing at the time this is created.